grc/core/FlowGraph.py

```diff
@@ -196,15 +196,17 @@ def renew_namespace(self):
         for expr in self.imports():
             try:
                 exec(expr, namespace)
-            except:
+            except Exception:
+                # This seems like a real bug...
```

Comment:
Do we have a logging capability in GRC to print this lalalalas?

Reply:
Logging for the exception has been added, still using a pass.